system-tests/lib/cre/don/jobs/jobs.go

```diff
@@ -30,6 +31,11 @@ func Create(offChainClient deployment.OffchainClient, don *devenv.DON, flags []s
 			defer cancel()
 			_, err := offChainClient.ProposeJob(ctx, jobReq)
 			if err != nil {
+				// Workflow specs get auto approved
+				// TODO: Narrow down scope by checking type == workflow
```

Comment:
pls create a JIRA ticket and link it here, won't let you off the hook that easily :D